code/modules/hallucinations/hallucinations.dm

```diff
@@ -24,6 +26,7 @@ GLOBAL_LIST_INIT(hallucinations, list(
 	HALLUCINATE_MAJOR = list(
 		/obj/effect/hallucination/abduction = 10,
 		/obj/effect/hallucination/assault = 10,
+		/obj/effect/hallucination/fake_grenade/spawner = 10,
```

Comment:
I feel like if this is a major hallucination, it should be defined in `major.dm`, to keep things organized.